network.go

```diff
@@ -469,3 +469,10 @@ func (n *network) getSvcRecords() []etchosts.Record {
 
 	return recs
 }
+
+func (n *network) scope() (driverapi.Scope, error) {
```

Comment:
Shall we change the word `Scope` to something else ?
I think the word `scope` can confuse folks for the service scope that is currently being discussed on the UX front.